packages/redux/src/products/actions/factories/fetchProductGroupingFactory.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  FETCH_PRODUCT_GROUPING_FAILURE,
+  FETCH_PRODUCT_GROUPING_REQUEST,
+  FETCH_PRODUCT_GROUPING_SUCCESS,
+} from '../../actionTypes';
+import { normalize } from 'normalizr';
+import { toError } from '@farfetch/blackout-client/helpers/client';
+import productSchema from '../../../entities/schemas/product';
+import type { Dispatch } from 'redux';
+import type {
+  GetProductGrouping,
+  GroupingQuery,
+  Product,
+  ProductGrouping,
+} from '@farfetch/blackout-client/products/types';
+
+/**
```

Comment:
I believe this doc block is not referencing the right function.
I had a similar issue on a previous [PR](https://github.com/Farfetch/blackout/pull/359/files#diff-6c5b873d0c4d39b5ed2522b401cf59c486bae4465670c38b2179df52e5774b78)

Reply:
You're 100% right, moved this doc block to be above the function it documents!